lms/djangoapps/course_home_api/dates/v1/views.py

```diff
@@ -74,7 +75,7 @@ def get(self, request, *args, **kwargs):
         course_key = CourseKey.from_string(course_key_string)
 
         if not course_home_mfe_dates_tab_is_active(course_key):
-            return Response(status=status.HTTP_404_NOT_FOUND)
+            raise Http404
```

Comment:
This is a switch for the dates tab waffle flag PR. This was a lingering comment.